nova_vm/src/heap/element_array.rs

```diff
@@ -2304,6 +2304,72 @@ impl ElementArrays {
                 .contains(&Some(element)),
         }
     }
+
+    /// This method creates a "shallow clone" of the elements of a trivial/dense array.
+    /// It does not do anything with descriptors and assumes there is a previous validation in place.
+    pub fn shallow_clone(&mut self, elements_vector: ElementsVector) -> SealableElementsVector {
```

Comment:
**note:** I think this function name is probably good as it is; my suggested renaming of the Array method does not transitively apply to this method :)